src/Solvers/batched_tridiagonal_solver.jl

```diff
@@ -0,0 +1,104 @@
+@hascuda using CUDAnative
+using GPUifyLoops: @launch, @loop, @unroll
+
+using Oceananigans: @loop_xy, device, array_type, launch_config
+
+"""
+    BatchedTridiagonalSolver
+
+A solver for batched tridiagonal systems.
```

Comment:
Could change this to "A batched solver for large numbers of triadiagonal systems."